live-examples/css-examples/opacity.html

```diff
@@ -0,0 +1,32 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="border-radius">
+
+    <div class="example-choice" initial-choice="true">
+        <pre><code id="example_one" class="language-css">opacity: 0;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true" data-clipboard-target="#example_one">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+    <div class="example-choice" initial-choice="true">
+        <pre><code id="example_two" class="language-css">opacity: 0.33;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true" data-clipboard-target="#example_two">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+    <div class="example-choice" initial-choice="true">
+        <pre><code id="example_three" class="language-css">opacity: 1;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true" data-clipboard-target="#example_three">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+
+</section>
+
+<div id="output" class="output large hidden">
+    <section id="default-example" class="default-example">
+      <div id="example-element-container">
+        <div id="example-element" class="transition-all">This is a text we might not see depending on the opacity.
```

Comment:
I think the text here is a bit misleading, as it suggested to me that it was the text we were setting `opacity` on. Maybe just "This is an element we might not see depending on the opacity." ?